File: app/src/main/java/io/treehouses/remote/pojo/ServicesData.java
package io.treehouses.remote.pojo;

import java.util.HashMap;
import java.util.List;

public class ServicesData {
    private List<String> available;
    private List<String> installed;
    private List<String> running;

    private HashMap<String, String> icon;
    private HashMap<String, String> info;
    private HashMap<String, String> autorun;


    public ServicesData() {}

    public List<String> getAvailable() {
        return available;
    }

    public List<String> getInstalled() {
        return installed;
    }

    public List<String> getRunning() {
        return running;
    }

    public HashMap<String, String> getIcon() {
        return icon;
    }

    public HashMap<String, String> getInfo() {
        return info;
    }

    public HashMap<String, String> getAutorun() {
        return autorun;
    }
}

File: app/src/main/java/io/treehouses/remote/pojo/ServicesData.kt
package io.treehouses.remote.pojo

import java.io.Serializable
import java.util.*


data class ServicesData(val available: List<String> = listOf(),
                        val installed: List<String> = listOf(),
                        val running: List<String> = listOf(),
                        val icon: HashMap<String, String> = hashMapOf(),
                        val info: HashMap<String, String> = hashMapOf(),
                        val autorun: HashMap<String, String> =hashMapOf()) : Serializable

